### ISSUE TITLE
Could you add batch mode support to the upcoming spring-cloud-stream-binder-kafka-reactive ?

### ISSUE THREAD

**Issue by davidmelia** [spring-cloud/spring-cloud-stream#2527]
Hi,

I've been testing the new spring-cloud-stream-binder-kafka-reactive 2022.0.0-M5 and have noticed that batch mode has not been implemented.  (i.e. org.springframework.cloud.stream.binder.ConsumerProperties.isBatchMode is not referenced in this binder but is referenced in the original spring-cloud-stream-binder-kafka).

Will this be implemented on full release ?

Thanks

**Comment by garyrussell** [comment]
No; the underlying project `reactor-kafka` has no mechanism for consuming in batch mode.

**Comment by garyrussell** [comment]
It was a new feature, recently added; reopening this now.
